verl/workers/megatron_workers.py

```diff
@@ -397,6 +397,10 @@ def compute_ref_log_prob(self, data: DataProto):
         torch.cuda.empty_cache()
         return output
 
+    @register(dispatch_mode=Dispatch.MEGATRON_COMPUTE_PROTO)
+    def compute_log_prob(self, data: DataProto):
+        raise NotImplementedError()
```

Comment:
To fix the missing `compute_log_prob` in megatron-lm backend, I think we can simply return an empty dataProto and print some warning.

As the generate sequences func in megatron_workers already recompute the log_prob.
See: https://github.com/volcengine/verl/blob/main/verl/workers/megatron_workers.py#L368-L373

Reply:
I'll fix it soon

Reply:
Shall we merge this PR first and fix compute log prob in a separate PR?

Reply:
Agree

Reply:
reverted the change and leave it for future PR